install/resources/mybb_theme.xml

```diff
@@ -4014,7 +4014,7 @@ var announcement_quickdelete_confirm = "{$lang->announcement_quickdelete_confirm
 	</td>]]></template>
 		<template name="forumdisplay_thread_rating_moved" version="1800"><![CDATA[<td class="{$bgcolor}" style="text-align: center;">-</td>]]></template>
 		<template name="forumdisplay_thread_unapproved_posts" version="1800"><![CDATA[ <span title="{$unapproved_posts_count}">({$thread['unapprovedposts']})</span>]]></template>
-		<template name="forumdisplay_threadlist" version="1820"><![CDATA[<div class="float_left">
+		<template name="forumdisplay_threadlist" version="1822"><![CDATA[<div class="float_left">
```

Comment:
Missed this version number
```suggestion
		<template name="forumdisplay_threadlist" version="1823"><![CDATA[<div class="float_left">
```

Reply:
I didn't touch the template `forumdisplay_threadlist` :(

Reply:
Probably want to just revert this change then, since the template doesn't seem changed apart from version.